Cargo.toml

```diff
@@ -289,6 +289,7 @@ itertools = "0.12.0"
 libc = "0.2.150"
 lscolors = { version = "0.16.0", default-features = false, features = [
   "nu-ansi-term",
```

Comment:
Is this still needed? At least the tests pass without it.

Reply:
probably not, thanks :)